src/assets/custom/css/_sass/_website-navigation.scss

```
@@ -25,3 +25,49 @@
   right: -8px;
   top: -10px;
 }
+
+.website-navigation__menu-item {
+  color: $grey--dark;
+}
+
+.website-navigation__menu-item:hover {
+  color: $coral;
+  text-decoration: none;
+}
+
+.has-sub-menu {
+  & > .website-navigation__menu-item::after {
+    content: "";
+    border: solid $grey--dark;
+    border-width: 0 2px 2px 0;
+    display: inline-block;
+    margin: 3px;
+    padding: 4.5px;
+    transform: rotate(45deg);
+  }
+  &:hover > .website-navigation__menu-item::after {
+    transform: rotate(225deg);
+    position: relative;
+    top: 7px;
+  }
+
+  &:hover > .sub-menu {
+    background-color: white;
+    display: block;
+    position: absolute;
+  }
+}
+
+.sub-menu {
+  display: none;
+}
```

Just a suggestion. For me this is a little more maintainable

```suggestion

.sub-menu {
  display: none;  
    
  .has-sub-menu:hover > & {
    background-color: white;
    display: block;
    position: absolute;
  }
}
```